Type-check fails due to invalid Cypress type import in auth constants

## Problem
yarn type-check fails during pre-commit because apps/web/cypress/support/auth-constants.ts imports a non-resolvable module:
 
- cypress/types/sinon-chai
 
## Root Cause
 TS2307: Cannot find module 'cypress/types/sinon-chai' or its corresponding type declarations.
 
## Scope
 apps/web/cypress/support/auth-constants.ts
 
## Fix
- Removed invalid import:
  - import sinonChai from "cypress/types/sinon-chai";
 

## Related
Follow-up/stacked with https://github.com/Greenstand/treetracker-wallet-app/pull/678 (same integration stream).

--- a/apps/web/cypress/support/auth-constants.ts
+++ b/apps/web/cypress/support/auth-constants.ts
@@ -1,7 +1,5 @@
 // 🚀 Shared Auth Constants & Helpers for 10x Development with Fixtures
 
-import sinonChai from "cypress/types/sinon-chai";
-
 // Static Selectors (don't change often, keep in constants)
 export const SELECTORS = {
   // Login selectors
@@ -98,7 +96,7 @@ export const expectFormFieldError = (message: string) => {
 export const generateUniqueEmail = () => `testuser${Date.now()}@example.com`;
 
 export const clearAuthSession = () => {
-  cy.window().then(win => {
+  cy.window().then((win) => {
     win.sessionStorage.clear();
     win.localStorage.clear();
   });

